README.md

```diff
@@ -124,24 +117,29 @@ Install with
 sudo make install
 ```
 
-Building from git
------------------
-
-You should have python2.7+ installed.
+## Building from git
 
 Download project with
 
 		git clone --recursive https://github.com/rockdaboot/libpsl
 		cd libpsl
 
-Build with GNU autotools
+### Building with GNU autotools
+
+Prerequisites:
+  - python2.7+ (for converting the PSL list)
+  - basic C development tools (compiler, linker)
+  - git, make, autoconf, autoconf-archive, autopoint, automake, autotools,
+  - libtool, gettext, m4, pkg-config
+  - for building docs: gtk-doc-tools (gtkdocize)
+  - for building tarballs: gzip, lzip
```

Comment:
building tarballs isn't part of building the project, it's part of maintaining the project. IMO it's not good material for README.md, but could be good material for RELEASING.md as project-internal release checklists etc.